tests/pr_test.go

```diff
@@ -93,7 +93,7 @@ func setupOptionsSecureSolution(t *testing.T, prefix string) *testhelper.TestOpt
 		"existing_kms_instance_guid": permanentResources["hpcs_south"],
 		"kms_endpoint_type":          "public",
 		"resource_group_name":        options.Prefix,
-		"name":                       options.Prefix,
+		"prefix":                     options.Prefix,
```

Comment:
Upgrade test is failing with this:
```
│ Error: Value for undeclared variable
│ 
│ A variable named "prefix" was assigned on the command line, but the root
│ module does not declare a variable of that name. To use this value, add a
│ "variable" block to the configuration.
```

Since prefix is optional, you could just change the test back to not use a prefix and it will pass. Or you can skip upgrade test

Reply:
Since other da's were passing the `prefix` in the pr_test instead of resource name so I kept it similar here.